README.md

```diff
@@ -57,9 +60,10 @@ That files are unique per self-hosted (**couldn't be shared between multiple ins
     -f carto-secrets.yaml \
     -f customization.yaml
```

Comment:
Should be adapted to `client-conf.yaml` or `client-customization.yaml` if we are going to provide it.

Reply:
after thinking about it, I don't think we need to provide it. And the doc says that you should create it. So I would leave it as it is, at least until we hear from real use